back/src/main/kotlin/com/sos/smartopenspace/domain/OpenSpace.kt

```diff
@@ -109,13 +110,17 @@ class OpenSpace(
   fun addTalk(talk: Talk): OpenSpace {
     checkIsFinishedQueue()
     checkIsActiveCallForPapers()
-    if (isTrackValid(talk.track))
-      throw NotValidTrackForOpenSpaceException()
+    checkTrackIsValid(talk.track)
     talk.openSpace = this
     talks.add(talk)
     return this
   }
 
+  private fun checkTrackIsValid(trac: Track?) {
+    if (isTrackValid(trac))
+      throw NotValidTrackForOpenSpaceException()
```

Comment:
quizas soy yo que no estoy entendiendo algo, pero no están un poco raros los nombres acá?

Queremos chequear que el track sea valido asi que hacemos un if isTrackValid. Hasta ahi venimos bien, pero despues tiramos un NOT Valid Track? No estabamos entrando en la parte del if donde el track sí es valido?

Reply:
Si, completamente raye mio jajajaja Ahora lo modifico